package.json

```diff
@@ -100,7 +102,7 @@
   "devDependencies": {
     "@emotion/babel-plugin": "^11.9.2",
     "@emotion/eslint-plugin": "^11.7.0",
-    "@netlify/plugin-nextjs": "^4.28.6",
+    "@netlify/plugin-nextjs": "^4.30.0",
```

Comment:
Not sure why this `@netlify/plugin-nextjs` update has to be part of this PR, but otherwise it is still ok.

Reply:
Build was failing https://github.com/Plant-for-the-Planet-org/planet-webapp/actions/runs/3927599948/jobs/6715360301